src/NuGet.Core/NuGet.Build.Tasks/NuGet.targets

```diff
@@ -61,6 +61,8 @@ Copyright (c) .NET Foundation. All rights reserved.
     <GetReferenceNearestTargetFrameworkTaskSupportsTargetPlatformParameter>true</GetReferenceNearestTargetFrameworkTaskSupportsTargetPlatformParameter>
     <!-- Flag if the Central package file is enabled -->
     <_CentralPackageVersionsEnabled Condition="'$(ManagePackageVersionsCentrally)' == 'true' AND '$(CentralPackageVersionsFileImported)' == 'true'">true</_CentralPackageVersionsEnabled>
+    <!-- Flag if the Private Asset Independent is enabled -->
+    <_PrivateAssetIndependentEnabled Condition="'$(PrivateAssetIndependent)' == 'true'">true</_PrivateAssetIndependentEnabled>
```

Comment:
Do we need this? It looks like in above line `_CentralPackageVersionsEnabled` used for some other type of calculation.